### DISCUSSION TITLE
Can you disable select rules on a select file?

### DISCUSSION THREAD

**Discussion by matthewmturner** [rome/tools#4583] [Q&A]
We are bringing in some external code that makes liberal use of `any` and so we see tons of errors. In the rest of our codebase we have been good about not using `any`.  So while we are reviewing the functionality I would like to hide the `any` errors in just these new files.  I do not want to disable at the workspace level or disable at each `any` location (there are a lot).

Thanks!

**Comment by ematipico** [comment]
At the moment we don't support this feature. You would need to suppress each violation
